arcgis-ios-sdk-samples/Features/Display feature layers/DisplayFeatureLayersViewController.swift

```diff
@@ -0,0 +1,189 @@
+// Copyright 2022 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class DisplayFeatureLayersViewController: UIViewController {
+    // MARK: Storyboard views
+    
+    /// The map view managed by the view controller.
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            // Initialize map with basemap.
+            mapView.map = AGSMap(basemapStyle: .arcGISTopographic)
+        }
+    }
+    
+    @IBOutlet var changeFeatureLayerBarButtonItem: UIBarButtonItem!
+    
+    // MARK: Instance properties
+    
+    var geodatabase: AGSGeodatabase!
+    var geoPackage: AGSGeoPackage?
+    
+    // MARK: Actions
+    
+    @IBAction func changeFeatureLayer() {
+        let alertController = UIAlertController(title: "Select a feature layer source", message: nil, preferredStyle: .actionSheet)
+        // Add an action to load a feature layer from a URL.
+        let featureServiceURLAction = UIAlertAction(title: "URL", style: .default) { (_) in
+            self.loadFeatureServiceURL()
+        }
+        alertController.addAction(featureServiceURLAction)
```

Comment:
No need to change: since the `UIAlertController.addAction(_:)` method is called 6 times here, we may use a `forEach` to save a few lines. But it really doesn't matter.